### DISCUSSION TITLE
quarto-mode package for emacs obsolete?

### DISCUSSION THREAD

**Discussion by sgutreuter** [quarto-dev/quarto-cli#5177] [Q&A]
When installing quarto-mode from melpa, I get the message "quarto-mode is an available _obsolete_ package".  I am guessing the the "obsolete" is bogus, because quarto-mode seems to install and work properly.

**Comment by cscheid** [comment]
Huh. I confess I don't know enough about melpa to understand what causes that. If there's something I can do on my end, I'd love to know about it so I can fix.
